templates/expo-template-tabs/__tests__/App-test.js

```diff
@@ -1,29 +1,35 @@
-import * as React from 'react';
-import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
-import renderer from 'react-test-renderer';
 
+import * as React from 'react';
+import { act, create } from 'react-test-renderer';
 import App from '../App';
 
 jest.mock('expo', () => ({
-  AppLoading: 'AppLoading',
+  Linking: {
+    makeUrl: () => '/',
+  },
+  SplashScreen: {
+    preventAutoHide: () => 'preventAutoHide',
+    hide: () => 'hide',
+  },
 }));
 
-jest.mock('../navigation/AppNavigator', () => 'AppNavigator');
 
+// test to prevent snapshot error here
+jest.mock('shortid', () => jest.fn(() => '1'))
+jest.mock("../navigation/BottomTabNavigator", () => "BottomTabNavigator");
+//
```

Comment:
What's with this random comment here?

Reply:
@TaylorAbraham that comment was brought in from the reproduction case i was using. I'll remove the comment shortly

Reply:
Fixed with the latest commit.